### ISSUE TITLE
NoClassDefFoundError: com.gitlab.mvysny.umn.core.MvstoreUMN$update$1

### ISSUE THREAD

**Issue by mvysny** [mvysny/aedict#848]
```
Fatal Exception: java.lang.NoClassDefFoundError: com.gitlab.mvysny.umn.core.MvstoreUMN$update$1
       at com.gitlab.mvysny.umn.core.MvstoreUMN.update(MvstoreUMN.kt:136)
       at com.gitlab.mvysny.umn.core.IndexingUMN.update(IndexingUMN.kt:412)
       at com.gitlab.mvysny.umn.core.LogRecord$DocumentUpdated.applyTo(AnnouncingUMN.kt:143)
       at com.gitlab.mvysny.umn.sync.GlobalLogPuller$1.invoke(SyncingUMN.kt:106)
       at com.gitlab.mvysny.umn.sync.GlobalLogPuller$1.invoke(SyncingUMN.kt:100)
       at com.gitlab.mvysny.umn.proxyclient.ProxyClient$Listener.onMessage(ProxyClient.kt:51)
       at okhttp3.internal.ws.RealWebSocket.onReadMessage(RealWebSocket.java:326)
       at okhttp3.internal.ws.WebSocketReader.readMessageFrame(WebSocketReader.java:221)
       at okhttp3.internal.ws.WebSocketReader.processNextFrame(WebSocketReader.java:105)
       at okhttp3.internal.ws.RealWebSocket.loopReader(RealWebSocket.java:273)
       at okhttp3.internal.ws.RealWebSocket$2.onResponse(RealWebSocket.java:213)
       at okhttp3.RealCall$AsyncCall.execute(RealCall.java:153)
       at okhttp3.internal.NamedRunnable.run(NamedRunnable.java:32)
       at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1113)
       at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:588)
       at java.lang.Thread.run(Thread.java:818)
```
Appears only on Galaxy Note4, Android 6.0.1. Please allow me to use this space to say how much I hate Android, its authors (who perverted the write-once-run-anywhere idea of Java into the fragmentation crap of Android), Samsung (not just for Note4, but for it's fucked-up `String.compare` because of which Aedict doesn't work on Tab3: #290 )

**Comment by mvysny** [comment]
Fuck! This crash is actually the retarded way of Android telling me I'm using `Map.computeIfPresent()` which is present only since Android level 24 but 6.0.1 is Android level 23.

**Comment by mvysny** [comment]
Fixed in Aedict 3.50.7
